app/options/apigw.cue

```diff
@@ -25,6 +25,14 @@ apigw: schema.#optionsGroup & {
 			type:        "bool"
 			description: "Enable extra logging"
 		}
+		profiler_enabled: {
```

Comment:
should this be enabled by default?

Reply:
> should this be enabled by default?
It should be on by default and global off, will act accordingly